### PR TITLE
Email reviewers on application submit

### PR DESCRIPTION
**Pull request by kristjanmik**
Adds a need seed for submissions.
Adds email lib that uses AWS SES.
Adds email templates.
Adds handlebars loader for webpack.

### CODE REVIEW

File: src/data/seed.js [hunk 1]
```diff
@@ -4,10 +4,16 @@ import {
...
+import {
+  up as seedSubmission,
+  down as flushSubmission,
+} from './seeders/submission';
+
 const up = async () => {
   console.log('Starting to seed');
   try {
     await seedUser();
+    await seedSubmission();
```
Comment (tryggvigy) [inline src/data/seed.js]: can these be done in parallel?
`await Promise.all([seedUser(), seedSubmission()])`

File: src/data/seed.js [hunk 2]
```diff
@@ -21,6 +27,7 @@ const down = async () => {
   console.log('Starting to flush');
   try {
     await flushUser();
+    await flushSubmission();
```
Comment (tryggvigy) [inline src/data/seed.js]: Same here

**Comment by tryggvigy** [comment]
LGTM!

File: src/data/seed.js (hunk 1 above)
Comment (kristjanmik) [reply src/data/seed.js]: Yes, great spot ;)

**Comment by tryggvigy** [comment]
new changes LGTM! 💯

**Comment by tryggvigy** [comment]
Really recommend this guy https://github.com/manosim/gitify, helping me know when you guys open PR's :)

**Pull request merged**